### ISSUE TITLE
A way of knowing a users monitor type (LCD or OLED)

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#2291]
### Is your feature request related to a problem? Please describe.

Right now it is hard to visually optimize content for LCD vs OLED for example my avatar is a little emissive on LCD headsets but when someone views my avatar from an OLED headset it is way too bright

### Describe the solution you'd like

I would like to be able to adjust things according to a users monitor type/headset display

### Describe alternatives you've considered

Optimizing for only OLED but this result in LED being more dull

### Additional Context

The glow on my avatar is driven to 3, 3, 3, 1 this enables my avatar to have a slight glow around it but when viewed from an OLED headset it is really bright

### Requesters

@epicEaston197

**Comment by ko-tengu** [comment]
There's literally no way to get this information and even across this there's a billion and a half other settings that would affect color reproduction such as headset brightness, windows color calibration, streamer choice and settings, mods, etc.

I will note that 3,3,3,1 is intended to be extremely bright and if it's a slight glow you may want to look at your own settings.

**Comment by epicEaston197** [comment]
>There's literally no way to get this information 
yeah and that's what I thought unfortunately but i decided i might not know the whole story so I just made an issue anyway another solution to this is possibly hard coding it to headset type

> I will note that 3,3,3,1 is intended to be extremely bright and if it's a slight glow you may want to look at your own settings.

my bloom settings are default
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/5b6a52be-1394-46b2-aebd-2771563146b4)
and I mean it is a slight glow
what I mean by this is the glow of the green is cascading on the black creating a green to black gradient
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/cfcb36ae-81fc-48c1-bbed-6bb5803871c9)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/9aa61963-9d5c-4c27-806b-d56608c457c2)

**Comment by JackTheFoxOtter** [comment]
This is not a optimization that should be done on user content level. If anything there might be a case for changing default settings depending on display type, but even that likely won't be practical because of the variance of different displays / configurations. If the bloom is too bright on someones display, it can be turned down. But as others have already pointed out, the display technology itself is only one of many factors that play into how something looks on screen. Most notably, it also depends on the displays configuration. Exposing this would likely lead to people building in special handling for separate display types, which at the end of the day would just make it harder for people to adjust their experience, since if you change your graphics settings to adjust for your display for the default shaders / conditions, now any user-made solution would be over- or undedercorrecting.

Your avatar is also kind of the worst-case for this, the perceived variance on PBR avatars or toon shaded avatars that don't have full saturation is going to be a lot less severe. Green is also the color the human visual system is most sensitive to (https://en.wikipedia.org/wiki/Color_vision), which makes this appear more intense than other colors would.

But there's also a privacy concern. Any piece of information about someone's hardware can be used for tracking purposes. There's already a bunch of information exposed, and it might not be a massive issue today, but I'd still be careful about exposing additional information.

**Comment by shiftyscales** [comment]
I do not feel this request is suitable. Ensuring colours look about the same is the responsibility of colour spaces / colour profiles- and the end-user is responsible for ensuring their displays are properly calibrated within a target space.

If you wish for the green to spill-over onto the black- as was already suggested- you should either texture it that way, or make use of a light source (or stick to using bloom knowing that some users may disable that post-processing effect locally). You are using an exceptionally bright colour so it is expected that it should appear excessively bright / vibrant- particularly on displays with a wider colour gamut / higher contrast like an OLED panel.
